Scroller/EditableWrapper.js

```diff
@@ -374,6 +378,30 @@ const EditableWrapper = (props) => {
 		}
 	}, [removeItem, editable.removeItemFuncRef]);
 
+	useEffect(() => {
+		// addEventListener to moveItems while scrolled
+		const container = scrollContentRef.current;
```

Comment:
There is scrollConatiner also, so the name can make confusion.
```suggestion
		const contentRef= scrollContentRef.current;
```

I'm not sure the name..